react/react1/week2/todo-app2/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import Header from './components/Header';
+import Timer from './components/Timer';
+import TodoLists from './components/TodoLists';
+import todoItems from './components/TodoItems';
+import './App.css';
+
+function App() {
+  const [todos, setTodos] = useState(todoItems);
+
+  const addingTodo = () => {
+    const newTodo = {
+      id: todos.length + 1,
```

Comment:
To generate a unique key, you can use this code (this is not my solution):
Math.random().toString(16).slice(2)

Because I caught a bug
![image](https://user-images.githubusercontent.com/73549460/197743029-1c476072-9d56-4f06-bd02-4a97b207affe.png)